src/components/RulesSelector.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { PropTypes } from 'react';
 import RuleLabel from './RuleLabel';
 
-const makeListItems = (filters, onRuleSelected) => {
-  return Object.keys(filters)
+const makeListItems = (filters, onRuleSelected) =>
```

Comment:
If you omit curly braces then it will be interpreted as returning an expression, or basically a single statement. It is an eslint thing, though